testdata/scripts/tiny.txt

```diff
@@ -0,0 +1,42 @@
+garble -debugdir=.obf-src build
+
+env TINY_PATTERN='^\/\/line :1$'
+env DEFAULT_PATTERN='^\/\/line \w\.go:[1-9][0-9]*$'
+env DEFAULT_STACK_PATTERN='^\t\w\.go:[1-9][0-9]*(\s\+0x[0-9a-f]+)?'
+env TINY_STACK_PATTERN='^\t\?\?:[0-9][0-9]*(\s\+0x[0-9a-f]+)?$'
+
+# Default mode
+
+# Check for file name leak protection
+grep $TINY_PATTERN .obf-src/main/main.go
+
+# Check for default line obfuscation
+grep $DEFAULT_PATTERN .obf-src/main/main.go
+
+! exec ./main$exe
+! stderr 'main\.go'
+! stderr $TINY_STACK_PATTERN
+stderr $DEFAULT_STACK_PATTERN
+
+[short] stop # no need to verify this with -short

```

Comment:
sorry, but this is backwards - `-short` should run `-tiny`, and the "long" test run should run both.